br/pkg/version/version.go

```diff
@@ -129,6 +129,24 @@ func CheckVersionForBackup(backupVersion *semver.Version) VerChecker {
 	}
 }
 
+// CheckVersionForBRPiTR checks whether version of the cluster and BR-pitr itself is compatible.
+// Note: BR'version > 6.1.0 at least in this function
+func CheckVersionForBRPiTR(s *metapb.Store, tikvVersion *semver.Version) error {
+	BRVersion, err := semver.NewVersion(removeVAndHash(build.ReleaseVersion))
+	if err != nil {
+		return errors.Annotatef(berrors.ErrVersionMismatch, "%s: invalid version, please recompile using `git fetch origin --tags && make build`", err)
+	}
+
+	// The versions of BR and TiKV should be the same, such as
+	// BR 6.1 <-> TiKV 6.1, BR 6.2 <-> TiKV 6.2
+	if BRVersion.Major != tikvVersion.Major || BRVersion.Minor != tikvVersion.Minor {
```

Comment:
what if 6.2 and 6.3 🤔？

Reply:
Does it mean that 6.2 and subsequent versions can be compatible with each other, but 6.1 cannot?

Reply:
> what if 6.2 and 6.3 🤔？

Does it mean that 6.2 and subsequent versions can be compatible with each other, but 6.1 cannot?